.github/actions/push-templates-json.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Push Templates JSON
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'templates.json'  # only trigger when this file changes
```

Comment:
Is this file going to be at the root of the directory? I think the path specified by `paths` is a path glob relative to root: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths

Reply:
Yes, it's right here: https://github.com/Shopify/extensions-templates/blob/main/templates.json